### DISCUSSION TITLE
Set different MinimumLevel to different sinks

### DISCUSSION THREAD

**Discussion by ceuser1** [serilog-contrib/serilog-sinks-grafana-loki#198] [General]
Scenario - two sinks, one for file and one for Loki. I would like to log everything to file (MinimumLevel=Verbose), and only important messages to Loki (MinimumLevel=Information). Is this possible?

If I set MinimumLevel=Verbose to LoggerConfiguration, then Loki will log everything. 
If I set MinimumLevel=Information to LoggerConfiguration, then File logger won't log anything below Information level.

I think we would need 'restrictedToMinimumLevel' flag in LokiSinkConfiguration?

**Comment by mishamyte** [comment]
Yes, correct. If you will set that flag, you will get the logs from the level you provided and higher

**Comment by ceuser1** [comment]
Do you have an example? I cannot find restrictedToMinimumLevel parameter in LokiHttp extension method, or in LokiSinkConfiguration class.

**Reply by mishamyte** [reply]
Parameter described in [wiki](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/wiki/Application-settings)
Here is the [line](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/blob/master/src/Serilog.Sinks.Grafana.Loki/LoggerConfigurationLokiExtensions.cs#L90) where it is in the configuration

Usage is typical

**Reply by ceuser1** [reply]
Thanks @mishamyte, works as intended. I used Serilog-Sinks-Loki instead of Serilog-Sinks-Grafana-Loki that's why I didn't find those parameters...
